inst/report/summary_report.Rmd

```diff
@@ -39,9 +39,9 @@ title: `r title`
 options(scipen=10000000)
 
 # Read files if hintr rds provided
-if(grepl("\\.rds$", params$outputs)) {
+if(grepl("\\.rds|qs$", params$outputs, ignore.case = TRUE)) {
```

Comment:
I don't think this regexp is checking exactly what you want. E.g.
```
> grepl("\\.rds|qs$", "hello.rds_ROB_IS_COOL")
[1] TRUE
```
```
> grepl("\\.rds|qs$", "hello_ROB_IS_COOL_qs")
[1] TRUE
```

I think you want something like:
```
grepl("(\\.rds$)|(\\.qs$)", params$outputs, ignore.case = TRUE)
```

Reply:
Oh thanks, I was trying to be clever here and use grepl instead of tools::file_ext as grepl is faster. But definitely more error prone :wink: and harder to read, and not really *that* much quicker. I've switched these over to use the built in helper.

Reply:
I know rob will change it but you know I love a good regex:
```
grepl("\\.(rds|qs)$", params$outputs, ignore.case = TRUE)
```